assets/js/components/game.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable import/extensions */
+import display from './display.js';
+import board from './board.js';
+import Player from './player.js';
+
+const game = (() => {
+  const initialTarget = document.querySelector('.card');
+  let turn = 0;
+  let playerXName;
+  let playerOName;
+  let players;
+  let result;
+  const winCombination = [[0, 1, 2], [3, 4, 5], [6, 7, 8],
+    [0, 3, 6], [1, 4, 7], [2, 5, 8],
+    [0, 4, 8], [2, 4, 6]];
+
+  const isEqual = (arr, target) => target.every((v) => arr.includes(v));
+  const winner = () => {
+    // let firstPlayerMove = board.returnBoard; closed in here
+    for (let k = 0; k < players.length; k += 1) {
+      for (let m = 0; m < winCombination.length; m += 1) {
+        console.log(isEqual(players[k].playersMove, winCombination[m]));
+        if (isEqual(players[k].playersMove, winCombination[m])) {
+          console.log('I am from winner');
+          return players[k];
+        }
+      }
+    }
+    return false;
+  };
+  const declareResult = () => {
+    document.querySelector('.status').style.display = 'block';
+    if (winner()) {
+      document.querySelector('.result').innerText = `Winner is: ${winner().name}`;
+    } else if (board.boardIsFull()) {
+      document.querySelector('.result').innerText = 'It is a Draw';
+    }
+  };
+  const checkResult = () => {
+    if (winner() || board.boardIsFull()) {
+      result = true;
+      declareResult();
+    }
+  };
+  const fillCell = (element) => {
+    if (!result) {
+      if (turn === 0) {
+        if (board.cellIsFilled(element)) {
+          document.getElementById(element).innerText = 'X';
+          document.querySelector('.player-0-panel').classList.remove('border-danger');
+          document.querySelector('.player-1-panel').classList.add('border-danger');
+          board.setCell(element, 'X');
+          players[0].playersMove.push(element);
+          turn = 1;
+        }
+      } else if (turn === 1) {
+        if (board.cellIsFilled(element)) {
+          document.getElementById(element).innerText = 'O';
+          document.querySelector('.player-1-panel').classList.remove('border-danger');
+          document.querySelector('.player-0-panel').classList.add('border-danger');
+          board.setCell(element, 'O');
+          players[1].playersMove.push(element);
+          turn = 0;
+        }
+      }
+    }
+    checkResult();
+  };
+  const loadGame = () => {
+    players = [Player(playerXName, 'X'), Player(playerOName, 'O')];
+  };
+  const showGrid = () => {
+    initialTarget.innerHTML = display.createTable()
+                              + display.createUserInfo(playerXName, playerOName);
+    // eslint-disable-next-line no-use-before-define
```

Comment:
All these rules should be specified in your `eslintrc.js` file if they are needed. Although you should learn to refactor your code to pass the check rather than by-passing it.